### PR TITLE
[Bug 14482] Added a new "solid" status bar style for iOS

### PR DESCRIPTION
**Pull request by livecodepanos**
'solid' style is opaque and automatically shifts down the app view by 20 pixels (i.e the height of the status bar)
Note that this patch is paired with an update in the IDE. More specifically, on stack:
Toolset/revstandalonesettings.rev --> card iOS --> button Status Bar Style --> Property Inspector
Add a new menuitem "Solid". This update is not yet pushed. I will push as soon as this pull request is reviewed.

### CODE REVIEW

File: engine/rsrc/mobile-device-template.plist [hunk 1]
```diff
@@ -91,6 +91,8 @@
 		${IPAD_STATUS_BAR_HIDDEN}
 	<key>UIStatusBarStyle</key>
 		${STATUS_BAR_STYLE}
+	<key>UIStatusBarSolid</key>
```
Comment (runrevmark) [inline engine/rsrc/mobile-device-template.plist]: I'd suggest checking in the Apple plist docs about what keys applications can put in them - I suspect Apple will reserve a certain amount of namespace and so using 'UIStatusBarSolid' might not be wise. I'm sure there is a custom/app-centric plist key namespace though :)

**Comment by livecodepanos** [comment]
Ok, I replaced "UIStatusBarSolid" with "com_livecode_StatusBarSolid" for now. In the future we could probably make custom key names use the app bundle prefix.

**Comment by runrevmark** [comment]
Great - looking good @runrevpanos. Just two things left...

A release note (make it quite explanatory as people might miss this feature otherwise).

Update the documentation entry for SetStatusBarStyle :)

**Comment by livecodepanos** [comment]
This pull request comes together with this one:
https://github.com/runrev/livecode-ide/pull/21

**Pull request merged**